security/pkg/k8s/controller/workloadsecret.go

```diff
@@ -188,6 +197,20 @@ func NewSecretController(ca ca.CertificateAuthority, requireOptIn bool, certTTL
 			UpdateFunc: c.scrtUpdated,
 		})
 
+	namespaceLW := listwatch.MultiNamespaceListerWatcher(namespaces, func(namespace string) cache.ListerWatcher {
```

Comment:
Since this pattern of using namespace labels to manage or not is common - maybe the code should be in pkg/ (if it's not already there ?). Would avoid more fragmentation and confusion...
@geeknoid

Reply:
sounds good to me, but might be done in a separate PR.